Fix warnings the old for_loops code

``` c++
for_loops_clad.cu: In function ‘double propagation(double, double, double, double, double, double, int, int, int)’:
/home/carli/dev/HASEonGPU/src/for_loops_clad.cu:356:1: warning: ‘forb_dump’ may be used uninitialized in this function [-Wmaybe-uninitialized]
         if (forb != 1){
 ^
for_loops_clad.cu:255:34: note: ‘forb_dump’ was declared here
     int tri_next, cell_z_next, forb, forb_dump;
                                  ^
for_loops_clad.cu:468:53: warning: ‘cell_z_next’ may be used uninitialized in this function [-Wmaybe-uninitialized]
    gain = gain * exp(N_tot*(beta_v[tri+cell_z*N_cells]*(sigma_e + sigma_a)-sigma_a)*length);
                                                     ^
for_loops_clad.cu:255:15: note: ‘cell_z_next’ was declared here
     int tri_next, cell_z_next, forb, forb_dump;
               ^
for_loops_clad.cu:468:43: warning: ‘tri_next’ may be used uninitialized in this function [-Wmaybe-uninitialized]
    gain = gain * exp(N_tot*(beta_v[tri+cell_z*N_cells]*(sigma_e + sigma_a)-sigma_a)*length);
                                           ^
for_loops_clad.cu:255:5: note: ‘tri_next’ was declared here
     int tri_next, cell_z_next, forb, forb_dump;
     ^

```

--- a/src/for_loops_clad.cu
+++ b/src/for_loops_clad.cu
@@ -257,6 +257,9 @@ double propagation(double x_pos, double y_pos, double z_pos, double x_dest, doub
     
     // is not used
     N_refl=0;
+    forb_dump = 0;
+    cell_z_next = 0;
+    tri_next = 0;
 
     if(N_refl){
       printf("_");

